server/index.js

```diff
@@ -111,14 +112,13 @@ module.exports = function start(options) {
                 var server = app.listen(app.get('port'), function() {
                     var host = server.address().address,
                         port = server.address().port;
-                    console.log('Styleguide server started at http://%s:%s', host, port)
```

Comment:
I like to leave this line in, as it provides feedback things are running (also, it's not related to the rest of this PR).